### PR TITLE
Pcap playbooks incident

### PR DESCRIPTION
**Pull request by David-BMS**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/25930

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by yaron-libman** [comment]
reviewed

File: Packs/PcapAnalysis/pack_metadata.json [hunk 1]
```diff
@@ -11,7 +11,9 @@
     ],
```
Comment (michalgold) [inline Packs/PcapAnalysis/pack_metadata.json:11]: Add use case: PCAP Analysis

File: Packs/PcapAnalysis/pack_metadata.json [hunk 2]
```diff
@@ -11,7 +11,9 @@
...
     "tags": [
         "pcap",
-        "mine"
+        "mine",
```
Comment (michalgold) [inline Packs/PcapAnalysis/pack_metadata.json:14]: remove "mine" please, you can add it to keywords instead.

File: Packs/PcapAnalysis/pack_metadata.json (hunk 1 above)
Comment (David-BMS) [reply Packs/PcapAnalysis/pack_metadata.json:11]: Fixed

File: Packs/PcapAnalysis/pack_metadata.json (hunk 2 above)
Comment (David-BMS) [reply Packs/PcapAnalysis/pack_metadata.json:14]: Done

**Pull request merged**